<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewMealDetails</class>
 <widget class="QDialog" name="ViewMealDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QComboBox" name="MealsComboBox">
     <item>
      <property name="text">
       <string>Meal Name 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Meal Name 2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="IngredientsLabel">
     <property name="text">
      <string>Ingredients:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="DirectionsLabel">
     <property name="text">
      <string>Directions:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QTextBrowser" name="DirectionsTextBrowser"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="CurrentDateLabel">
     <property name="text">
      <string>Current Date</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QListWidget" name="listOfIngredients"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ViewMealDetails</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ViewMealDetails</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
